### PR TITLE
Acceleration fine tuning + minor fixes

### PR DESCRIPTION
**Pull request by kmatheussen**

### CODE REVIEW

**Comment by kmatheussen** [comment]
With this PR, it's a lot safer to accelerate. Perhaps accelerate more often when there is a lot of food? (I don't understand that code)

**Comment by ermiyaeskandary** [comment]
Why has the default PR template not been used ? It's there for a reason - so we know things we need to know. What is the purpose of this ?

**Comment by kmatheussen** [comment]
It's a bunch of minor fixes. I originally only wanted to avoid accelerating unnecessary, but there is a lot of bad code here, and my frustration level got too high to fill in that form. Just look at this code:

```
 getFood: function() {
            // Filters the nearest food by getting the distance
            return window.foods.filter(function(val) {
                return val !== null && val !== undefined;
            }).map(canvas.setDistanceFromSnake).filter(function(val) {
                var isInsideDangerAngles = canvas.isInsideAngle(val, window.snake.ang - 3 * Math.PI / 4, window.snake.ang - Math.PI / 4);
                isInsideDangerAngles = isInsideDangerAngles || canvas.isInsideAngle(val, window.snake.ang + Math.PI / 4, window.snake.ang + 3 * Math.PI / 4);
                return !(isInsideDangerAngles && (val.distance <= 22500));
            });
        },
```

It would be better if you were stricter on code quality than on pull request description.

**Comment by kmatheussen** [comment]
The point is that without this PR, the snake accelerates first, and in the next update checks if there are snakes nearby so that we have to stop accelerating. With this PR, we check if there are snakes nearby before accelerating, and we probably don't die as often.

**Comment by ermiyaeskandary** [comment]
@kmatheussen 

> It would be better if you were stricter on code quality than on pull request description.

Me and @ChadSki are exactly working on code style and quality linting - if you were in our Gitter room, you would have know. The thing is you need to have enough compliance with the less important stuff like PR templates, to progress further and further...

**Comment by ermiyaeskandary** [comment]
@kmatheussen 

> This branch is out-of-date with the base branch

**Comment by kmatheussen** [comment]
This is a tiny program with a lot of bad code. It makes very little sense making people fill out a form when creating a pull request.

**Comment by ermiyaeskandary** [comment]
@kmatheussen There still needs to be a standard to follow...

**Comment by kmatheussen** [comment]
For the current state of the program, I would say it's enough to only request a lose description of what the pull request does, or just a headline. The current form is overwhelming, and unnecessary.

**Comment by ermiyaeskandary** [comment]
Well, I don't and I don't think the rest think it is overwhelming as well. Some people do about 30 commits and then just want to summarize it in one headline ?!

**Comment by ChadSki** [comment]
:+1: Thanks for the cleanup effort @kmatheussen!

30 purposeful commits with clear labels is okay in my book. In this case, each commit fixes a particular thing.

What irks me is when there are dozens of commits without clear labels or purpose, undoing and redoing work. That's when it becomes hard to follow along.

Once we have good linting set up, someone will have to go through and clean up the codebase to match. Then it will look clean and stay clean.

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/136/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Comment by ermiyaeskandary** [comment]
Yeah - sure. I was just saying some people might have about 40 commits with unclear labels and it cannot be summarized into a single title. I want referring to @kmatheussen. General statement . I 100% acknowledge the work @kmatheussen has done here. For the codebase thing, Hound is fixed now and I'll go through the code base linting :) as Hound just comments on PRs not all of the code base. Great work @kmatheussen :+1:
